docs/src/examples/Gol, Lazar & Belta (2013).jl

```diff
@@ -56,15 +54,15 @@ transition_cost = QuadraticControlFunction(ones(1, 1))
 problem = OptimalControlProblem(
     system,
     q0, x0,
-    Fill(state_cost, N),
-    Fill(Fill(transition_cost, ntransitions(system)), N),
+    Fill(state_cost, N), 
```

Comment:
Remove trailing space at the end of the line

Reply:
Sure!